tests/foreman/longrun/test_inc_updates.py

```diff
@@ -36,109 +35,104 @@
 
 
 @pytest.fixture(scope='module')
-def module_lce_library(module_entitlement_manifest_org):
+def module_lce_library(module_sca_manifest_org):
```

Comment:
Should we import `module_target_sat` here, and into related fixtures currently using `entities` ?

```suggestion
def module_lce_library(module_sca_manifest_org, module_target_sat):
```

Reply:
this will consider separately